user-service/src/main/java/com/innowise/userservice/exception/AppRuntimeException.java

```diff
@@ -0,0 +1,11 @@
+package com.innowise.userservice.exception;
+
+import java.io.Serializable;
+
+public abstract class AppRuntimeException extends RuntimeException implements Serializable {
```

Comment:
all your exceptions are quite heavy
never saw such logic for them
keep in mind that depends on project exceptions can be just simple default ones with RuntimeException inherited constructors